frontend/utils/auth.utils.ts

```
@@ -0,0 +1,54 @@
+  return api.post({ email, password }, "/api/v1/accounts/users/login/");
+};
+
+const logout = (token: string, logout_all: boolean = false) => {
+  if (logout_all) {
+    return api.post(
+      { Authorization: "Token " + token },
+      "/api/v1/accounts/users/logout-all/"
+    );
+  }
+  return api.post(
+    { Authorization: "Token " + token },
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)